WooCommerce/WooCommerceTests/ViewRelated/Coupons/AddEditCouponViewModelTests.swift

```diff
@@ -0,0 +1,34 @@
+import XCTest
+@testable import Yosemite
+@testable import WooCommerce
+
+final class AddEditCouponViewModelTests: XCTestCase {
+
+    func test_titleView_property_return_expected_values_on_creation_depending_on_discountType() {
+        let viewModel1 = AddEditCouponViewModel(siteID: 123, discountType: .percent)
+        XCTAssertEqual(viewModel1.titleView, "Create percentage discount")
+
+        let viewModel2 = AddEditCouponViewModel(siteID: 123, discountType: .fixedCart)
+        XCTAssertEqual(viewModel2.titleView, "Create fixed card discount")
+
+        let viewModel3 = AddEditCouponViewModel(siteID: 123, discountType: .fixedProduct)
+        XCTAssertEqual(viewModel3.titleView, "Create fixed product discount")
+
+        let viewModel4 = AddEditCouponViewModel(siteID: 123, discountType: .other)
+        XCTAssertEqual(viewModel4.titleView, "Create discount")
```

Comment:
❓ When testing localized string, do we need to compare the content with localized string instead? I think this could fail when run on a device with a different language - although it's not really important given that our CI runs with English and US locales as default settings.